src/dual.jl

```diff
@@ -173,6 +173,8 @@ Base.isinteger(n::Dual) = isinteger(value(n))
 Base.iseven(n::Dual) = iseven(value(n))
 Base.isodd(n::Dual) = isodd(value(n))
 
+Base.rtoldefault{N, T <: Real}(::Type{Dual{N,T}}) = Base.rtoldefault(T)
```

Comment:
Maybe move this to [here](https://github.com/tpapp/ForwardDiff.jl/blob/f90357d9d6a30789930be0832c109153b86d6d83/src/dual.jl#L102) (so that it's not under the "predicates" section, since it's not a predicate).